sys-kernel/installkernel-systemd-boot/files/installkernel-systemd-boot-2-r2-00-00machineid-directory.install

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
don't we want `sh` compatibility for these scripts?

Reply:
I guess we do, apart from the dracut scripts all the other scripts in this directory use sh instead of bash.

Fixed